src/components/SlideButton/SlideButton.js

```diff
@@ -0,0 +1,147 @@
+import { ChevronsRight } from '@lana/b2c-mapp-ui-assets';
+
+const components = {
+  ChevronsRight,
+};
+
+const sliderStartClass = 'started';
```

Comment:
Please remove these two lines and line: 36, below in favor of using a `:class="{ started: (isStarted && !isCompleted), completed: isCompleted }"` binding where the classnames are only stored inside the markup. Let me know if you'd like to discuss this further or if you'd like any help.